python/ccf/ledger.py

```diff
@@ -173,10 +192,46 @@ def __next__(self):
 
 class Ledger:
 
-    _filename = None
+    _filenames = []
+    _fileindex = 0
+
+    def __init__(self, name: str):
```

Comment:
The `governance_test` on the CI (see [here](https://dev.azure.com/MSRC-CCF/CCF/_build/results?buildId=10713&view=logs&j=ee9484ec-b476-577b-aa0e-d5ddfcbcd149&t=e2c1a4cf-16d0-5038-f9f6-343f9806d38b&l=700)) is still failing because of an edge case. In the test, there is only one chunk at this point which is not yet committed. Hence, `len(self._filenames)` is `0`.

Would you mind updating the constructor of `Ledger` slightly so that it works with this edge case too? I also think the `if os.path.isdir(name)` check is no longer required as we always expect `name` to be a directory.

Reply:
Updated now so that even files that are not .committed can be used, but logging a warning just in case.